examples/error/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>St. Jude Cloud Error</title>
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.3.1/css/all.css"
+      integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="../../dist/css/stjude-cloud-theme.css"
+    />
+  </head>
+
+  <body>
+    <div class="d-flex">
+      <div class="sjc-error">
+        <img src="../../dist/images/404.png" class="error-image" />
+        <div class="error-list-container">
+          <h2 class="error-header">Page Not Found</h2>
+          <p class="error-subheader">Here are some helpful links instead.</p>
+          <div class="error-links-list">
+            <a href="https://www.stjude.cloud/data">Request Data</a>
```

Comment:
We should remove the links to Data, Tools and Visualizations. Maybe stjude.cloud homepage, Documentation and Contact Us?

Reply:
I think you're right. We should update this. 
<img width="448" alt="Screen Shot 2020-03-27 at 11 43 31 AM" src="https://user-images.githubusercontent.com/2433649/77779245-42c9b080-7020-11ea-99dc-2f94224e0290.png">

Thoughts?

Reply:
Great! I forgot about the button also going to the homepage. Maybe take off the first homepage instance or replace it with links to each of the app homepages then just have the button go to the homepage?

Reply:
Keep in mind that one links to `https://stjude.cloud/`, the other links to `/`. Maybe a wording change?